.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Why did you require this?

Reply:
it was auto generated by rspec while installation, purpose of this is to maybe directly execute commands from here instead of command line, in my case I should also add require rails helper then everything that needs rails helper and spec helper will be pre loaded